pkg/plugin/test-keeper/plugin/comment_message.go

```diff
@@ -0,0 +1,61 @@
+package plugin
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/arquillian/ike-prow-plugins/pkg/github"
+	"github.com/arquillian/ike-prow-plugins/pkg/scm"
+	"github.com/arquillian/ike-prow-plugins/pkg/utils"
+)
+
+const (
+	beginning = "There wasn't found any test file that would be changed in this PR. Please add a test case to this PR.\n" +
```

Comment:
You can probably use \` \` for better formatting of those messages. Now it's quite hard to review them as GH doesn't wrap it.

Reply:
I will review those separately and come back with proposal.

Reply:
using `` won't help much as in some cases I cannot include the new line that this string would keep.
I have split it so it is more readable

Reply:
934ae04